src/Clave.Expressionify.Generator/ExpressionifySourceGenerator.cs

```diff
@@ -59,7 +61,11 @@ private static void Execute(GeneratorExecutionContext context, IEnumerable<Metho
 
                 foreach (var (path, source) in replacedTypes)
                 {
-                    context.AddSource(Path.GetFileNameWithoutExtension(path)+$"_expressionify_{i++}.cs", SourceText.From(source.ToFullString(), Encoding.UTF8));
+                    var sourceCode = NullableDirective + source.ToFullString();
```

Comment:
Note: I'm adding `#nullable enable` to all generated sources, as otherwise the generated code won't compile with the error `CS8669: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context. Auto-generated code requires an explicit '#nullable' directive in source.` (which is a warning, but we have WarningsAsErrors enabled)

Not sure, if this is fine for "legacy" projects where nullable is disabled, as I don't have any bigger old project to test it in, where we also use Expressionify...

Reply:
I'm also not fully sure about old projects, but from all I could find it seemed fine.